introduction.rst

```diff
@@ -0,0 +1,43 @@
+############
+Introduction
+############
+
+Welcome to Network4Dev! Network4Dev is a community-driven, open source 
+networking site. The site was founded by 
+`Daniel Dib <https://www.linkedin.com/in/danieldib/>`_ in 2018 and all 
+work behind the scenes is done by 
+`Cristian Sirbu <https://www.linkedin.com/in/sirbucristian/>`_. At the
+time of this writing ``2018-08-19``, the two maintainers of the
+`GitHub repository <https://github.com/network4dev/n4d>`_ is 
+Cristian and Daniel.
+
+As infrastructure is moving towards more elastic, composable and
+automated setups, infrastructure engineers must have a broader
+set of skills while still remaining relevant within their speciality.
+This means that networking people must understand compute, storage,
+applications, systems and coding at a level where they can interact
+with the other teams. Conversely, it means that people working
+mainly with systems and apps must have some understanding of
+networking. While there are plenty of sites and courses for
+networking people to learn about automation, there are a lot less
+resources for people working with systems and apps to learn about
+networking. The goal of N4D is to fill this gap.
+
+In order to write for people that don't have a strict networking
+background, the articles will be written to be short, concise and
+to the point. The articles will not be deep dives but instead
+focus on fundamentals and the **WHY** of networking.
+
+By making this site open source and community driven, more people
+can contribute into making this site living and more frequently
+updated than a site where only one or a couple of people are
+contributing.
+
+In order to contribute to this site, read the :doc:`contributing`
+guide.
+
+Please note that all contributions to this site will be reviewed
+for quality both in terms of language, technical level in order
+to preserve high quality content.
+
```

Comment:
Suggest slight rephrasing: "Please note that all contributions to this site will be reviewed and edited by the maintainers for quality, accuracy and clarity both in terms of language and technical level in order to maintain our goals."

Reply:
Have updated this phrase.